capella2polarion/converters/element_converter.py

```diff
@@ -433,6 +457,7 @@ def __generic_work_item(
             uuid_capella=obj.uuid,
             description=polarion_api.HtmlContent(value),
             status="open",
+            layer=layer,
```

Comment:
I thought we would add the layer optional via the `add_attributes` serializer as we often already have the layer as part of the work item type